docs/resources/Guild.md

```diff
@@ -644,6 +648,14 @@ In guilds with [Membership Screening](https://support.discord.com/hc/en-us/artic
 > warn
 > We are making significant changes to the Membership Screening API specifically related to getting and editing the Membership Screening object. Long story short is that it can be improved. As such, we have removed those documentation. There will **not be** any changes to how pending members work, as outlined above. That behavior will stay the same.
 
+### Inventory Settings Object
+
+###### Inventory Settings Structure
+
+| Field                      | Type    | Description                                  |
+| -------------------------- | ------- | -------------------------------------------- |
+| is_emoji_pack_collectible? | boolean | Allows everyone to use emojis across servers |
```

Comment:
that sounds a bit misleading, it only applies to nitro users with emojis of this specific server, not everyone

Reply:
it's not nitro only, but users need to collect the pack and add it to their inventory

Reply:
wait what? they're letting go of the emojis nitro perk or?

Reply:
you can collect one emoji pack for free and you'll need to upgrade to nitto to collect more (up to 200)